std/exception.d

```diff
@@ -1478,6 +1478,34 @@ private bool isUnionAliasedImpl(T)(size_t offset)
         static assert( isUnionAliased!(S.A5, 1)); //a5.b1;
 }
 
+package string errnoString(int errno)
```

Comment:
See https://issues.dlang.org/show_bug.cgi?id=13567. The [docs](https://dlang.org/spec/function.html#function-attribute-inference) need to be updated, since https://github.com/dlang/dmd/pull/1877 was merged.